kiwi/package_manager/microdnf.py

```diff
@@ -102,7 +102,16 @@ def process_install_requests_bootstrap(self, root_bind=None):
         bash_command = [
             'microdnf'
         ] + ['--refresh'] + self.dnf_args + [
-            '--installroot', self.root_dir
+            '--installroot', self.root_dir, '--noplugins',
+            '--setopt=cachedir={0}'.format(
+                self.repository.shared_dnf_dir['cache-dir']
```

Comment:
Does `self.repository` exist at this point?

Reply:
sure, the package manager object can only be created by passing in a repository instance

Reply:
Ah, okay.

Reply:
```python
class PackageManagerMicroDnf(PackageManagerBase):
```

and the base class has

```python
def __init__(self, repository, custom_args=None):
```